app/services/listing_index_service/search/zappy_adapter.rb

```diff
@@ -0,0 +1,84 @@
+module ListingIndexService::Search
+
+  class ZappyAdapter < SearchEngineAdapter
+
+    INCLUDE_MAP = {
+      listing_images: :listing_images,
+      author: :author,
+      num_of_reviews: {author: :received_testimonials},
+      location: :location
+    }
+
+    def initialize
+      @conn = Faraday.new(url: "http://127.0.0.1:8080") do |c|
+         c.request  :url_encoded             # form-encode POST params
+         c.response :logger                  # log requests to STDOUT
+         c.response :json, :content_type => /\bjson$/
+         c.adapter  Faraday.default_adapter  # make requests with Net::HTTP
+         c.use Faraday::Response::RaiseError if Rails.env.development?
```

Comment:
Hmm... I'm just wondering that this would maybe be a bit cleaner if you'd pass `raise_errors` as on argument to the `initialize` method instead of being dependant on the outside world state.